<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarlaControlW</class>
 <widget class="QMainWindow" name="CarlaControlW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carla Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="w_plugins" native="true">
      <layout class="QVBoxLayout" name="layout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="act_file_connect"/>
    <addaction name="act_file_refresh"/>
    <addaction name="separator"/>
    <addaction name="act_file_quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="act_help_about"/>
    <addaction name="act_help_about_qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_file_connect"/>
   <addaction name="act_file_refresh"/>
   <addaction name="separator"/>
   <addaction name="act_file_quit"/>
  </widget>
  <action name="act_file_connect">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/16x16/network-connect.png</normaloff>:/16x16/network-connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect...</string>
   </property>
  </action>
  <action name="act_file_quit">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/16x16/application-exit.png</normaloff>:/16x16/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="act_help_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="act_help_about_qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="act_file_refresh">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/16x16/view-refresh.png</normaloff>:/16x16/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_file_quit</sender>
   <signal>triggered()</signal>
   <receiver>CarlaControlW</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
